src/main/java/com/meilisearch/sdk/IndexesHandler.java

```diff
@@ -53,21 +55,41 @@ TaskInfo create(String uid, String primaryKey) throws MeilisearchException {
      * Gets an index from its uid
      *
      * @param uid Unique identifier of the index to get
-     * @return Meilisearch API response
+     * @return Meilisearch API response as Index instance
      * @throws MeilisearchException if an error occurs
      */
-    String get(String uid) throws MeilisearchException {
-        String requestQuery = "/indexes/" + uid;
-        return httpClient.get(requestQuery, String.class);
+    Index getIndex(String uid) throws MeilisearchException {
+        return httpClient.get(new IndexesQuery().toQuery(uid), Index.class);
     }
 
     /**
      * Gets all indexes in the current Meilisearch instance
      *
-     * @return Meilisearch API response
+     * @return Results containing a list of indexes
      * @throws MeilisearchException if an error occurs
      */
-    String getAll() throws MeilisearchException {
+    Results<Index> getIndexes() throws MeilisearchException {
+        return httpClient.get("/indexes", Results.class, Index.class);
```

Comment:
It could be nice to use `IndexesQuery` everywhere so that we only hardcode `/indexes` in `toQuery`. Nonetheless, maybe it is worth creating an issue for that change to avoid making this PR more complex

Reply:
We can indeed do this but it means creating a new function that will require resources and complicate the general call such as:
`return httpClient.get("/indexes", Results.class, Index.class);`
will became:
`return httpClient.get(new IndexesQuery().toQuery(), Results.class, Index.class);`

Reply:
I prefer the second option for consistency, but you are maybe right. You can ignore this for the moment but I would like the opinion of @brunoocasali on that

Reply:
My initial conception around `IndexesQuery`, `KeysQuery`, and so on, where to handle query parameters. If we are very strict, we can say that the `/indexes` path is related to the URL, not the **query parameters**, so it shouldn't be inside.

In any case, I don't think `/indexes` should be treated as handled inside of `IndexesQuery`, but if we use a `private constant` inside of `IndexesHandler` is enough.

If the path change in the future, we will have two places to change the tests, `IndexesQuery` and `IndexesHandler`.